Developer Samples/Wpf.ObjectAsStateInvoice/Wpf.ObjectAsStateInvoice/Wpf.ObjectAsStateInvoice.csproj

```diff
@@ -39,12 +39,57 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="ActiproSoftware.Docking.Wpf, Version=13.2.592.0, Culture=neutral, PublicKeyToken=36ff2196ab5654b9, processorArchitecture=MSIL">

```

Comment:
are these explicit paths/references correct? it still appears to be pointing at net461 for these libs (and an older inrule version at that) - is that due to not having net472 packages available yet?

Reply:
will have to check, everything was auto nuget upgraded, maybe it doesn't have a new version?  will check...